src/shogun/neuralnets/DeepAutoencoder.cpp

```diff
@@ -321,6 +321,5 @@ void DeepAutoencoder::init()
 	    "Pre-training L1 regularization coeff");
 	SG_ADD(&m_do_pretrain, "do_pretrain",
 	    "Whether to pretrain with relevant parameters");
-
```

Comment:
in the future, please don't add such whitespace changes to your commit

Reply:
Thanks for pointing this out, I overlooked it while committing